src/assets/data/contributors.js

```diff
@@ -0,0 +1,75 @@
+/*
+  MIT License
+
+  Copyright (c) 2024 Open Source  UOM
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy
+  of this software and associated documentation files (the "Software"), to deal
+  in the Software without restriction, including without limitation the rights
+  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  copies of the Software, and to permit persons to whom the Software is
+  furnished to do so, subject to the following conditions:
+
+  The above copyright notice and this permission notice shall be included in all
+  copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+  SOFTWARE.
+
+  Made by Open Source UoM (https://opensource.uom.gr)
+
+  Project members:
+    -Apostolidis
+    -Davios
+    -Iosifidis
+    -Konstantinidis
+    -Mpakalis
+    -Nasis
+    -Omiliades
+    -Patsouras
+    -Fakidis
+
+*/
+
+import i18n from "../../i18n";
+
+const Contributors = [
```

Comment:
As i told you in the issue, we need to take these data from the GitHub API :

- https://contrib.rocks/image?repo=Open-Source-UoM/MyUoM

or 

- https://stackoverflow.com/questions/48874412/github-api-how-to-get-the-top-contributors-sorted-for-a-given-repository

Reply:
@GeorgeApos How can we achieve the following style?

![image](https://github.com/open-source-uom/myuom/assets/91419219/c2e29061-1289-4e27-a7c8-855032c9ca20)


or you want this style?

![Uploading image.png…]()

Reply:
I like the style , however we can't support updating the Contributors data so we wan't to retrieve them through the Github API

Reply:
@GeorgeApos I was trying to fetch contributors then I got the following error while fetching

![image](https://github.com/open-source-uom/myuom/assets/91419219/cf330617-2c30-4552-94b5-f1ca8e3515e9)

Reply:
@GeorgeApos I consumed Github API to display Contributors list